src/wp-admin/includes/nav-menu.php

```diff
@@ -678,50 +751,68 @@ function wp_nav_menu_item_taxonomy_meta_box( $object, $box ) {
 		'_wpnonce',
 	);
 
+	$most_used_url = $view_all_url = $search_url = '';
+	if ( $nav_menu_selected_id ) {
+		$most_used_url = esc_url( add_query_arg( $taxonomy_name . '-tab', 'most-used', remove_query_arg( $removed_args ) ) );
+		$view_all_url  = esc_url( add_query_arg( $taxonomy_name . '-tab', 'all', remove_query_arg( $removed_args ) ) );
+		$search_url    = esc_url( add_query_arg( $taxonomy_name . '-tab', 'search', remove_query_arg( $removed_args ) ) );
+	}
```

Comment:
Is this same code as above? or in a different function?

Reply:
Same /similar code, but in a different function.